apps/emqx_utils/src/emqx_utils_stream.erl

```diff
@@ -22,6 +22,8 @@
     list/1,
     mqueue/1,
     map/2,
+    filter/2,
+    foreach/2,
```

Comment:
Tiny nit. Move to `%% Evaluating` section?

Reply:
✅